features/step_definitions/take_part_pages_steps.rb

```diff
@@ -1,9 +1,9 @@
 Given(/^there are some take part pages for the get involved section$/) do
```

Comment:
Not a blocking review, but the naming here is a bit confusing: `page1`/`page2`/`page3` but ordering `2`, `3`, `1`. Could the names be made more clear regarding how they're arranged, perhaps?

Reply:
I completely agree - was going to attempt a refactor, but didn't get around to it. Not wanting to stall this any longer (I'm on leave until Wednesday), going to merge this for now, maybe this entire code will be deleted before a refactor is due.